tests/gis_tests/gdal_tests/test_raster.py

```diff
@@ -309,15 +309,33 @@ def setUp(self):
         rs = GDALRaster(self.rs_path)
         self.band = rs.bands[0]
 
+    def tearDown(self):
```

Comment:
I think usually we use a try/finally block if this is only applicable to 1 test.